js/views/Settings.js

```diff
@@ -0,0 +1,43 @@
+import { View } from 'backbone';
+import loadTemplate from '../utils/loadTemplate';
+
+export default class Settings extends View {
```

Comment:
Sorry... I know you were just following my pattern, but I've realized we don't need to specify the class name (since we're not using it anywhere by name). So, the `Settings` could just be removed from this line.